pkg/nodebootstrap/legacy/ubuntu.go

```diff
@@ -59,7 +59,13 @@ func (b UbuntuBootstrapper) UserData() (string, error) {
 }
 
 func makeUbuntuConfig(spec *api.ClusterConfig, ng *api.NodeGroup) ([]configFile, error) {
-	clientConfigData, err := makeClientConfigData(spec, kubeconfig.HeptioAuthenticatorAWS)
+	authenticator := kubeconfig.HeptioAuthenticatorAWS
+	// Ubuntu 20.04 doesn't ship anymore heptio
+	if ng.AMIFamily == api.NodeImageFamilyUbuntu2004 {
```

Comment:
That's not true. The 1.20 images do no longer ship heptio. The 1.19 images (also 20.04 based) do ship heptio.

But all latest images Ubuntu ships have a recent enough `aws-cli` version so we can unconditionally switch to `AWSEKSAuthenticator`

Reply:
Oh. Thanks for clarifying this! I updated this PR to reflect what you said.